File: src/main/java/ru/trukhmanov/twochairsbackend/repository/game/DeckQuestionRepository.java
package ru.trukhmanov.twochairsbackend.repository.game;

import org.springframework.data.jpa.repository.JpaRepository;
import ru.trukhmanov.twochairsbackend.entity.DeckQuestion;
import ru.trukhmanov.twochairsbackend.entity.DeckQuestionId;

public interface DeckQuestionRepository extends JpaRepository<DeckQuestion, DeckQuestionId> {
    boolean existsByIdDeckIdAndIdQuestionId(long deckId, long questionId);
}

File: src/main/kotlin/ru/trukhmanov/twochairsbackend/repository/game/DeckQuestionRepository.kt
package ru.trukhmanov.twochairsbackend.repository.game

import org.springframework.data.jpa.repository.JpaRepository
import ru.trukhmanov.twochairsbackend.entity.DeckQuestion
import ru.trukhmanov.twochairsbackend.entity.DeckQuestionId

interface DeckQuestionRepository : JpaRepository<DeckQuestion, DeckQuestionId> {
    fun existsByIdDeckIdAndIdQuestionId(deckId: Long, questionId: Long): Boolean

    fun deleteByIdDeckIdAndIdQuestionId(deckId: Long, questionId: Long)
}
